content/digital_design/puzzle_edgedetect/_index.en.md

```diff
@@ -0,0 +1,22 @@
+---
+title: Logic Puzzle - Edge Detect
+tags: [puzzle]
+description: "Detect the rising and falling edge of a signal"
+weight: 30
+---
+
+In this puzzle, you will learn how to detect an instantaneous change in a signal.
+
+In the world of digital circuits, these changes will correspond to either a rising edge (i.e. signal going from low to high) or a falling edge (i.e. signal going from high to low). Detecting rising and falling edges is extremely important for digital circuits. For example, did a user just press a button? Or did they just release a button?
+
```

Comment:
here would be great to have a little interactive demo like Thea does at https://blog.thea.codes/the-design-of-the-juno-dco/
![image](https://user-images.githubusercontent.com/634220/197422214-ff3a24aa-254f-4c80-840e-21a76d43f424.png)